[FIX] update definition acq_time for sessions.tsv (#986)
bids-standard/bids-specification: src/03-modality-agnostic-files.md

{{ MACROS___make_columns_table(
   {
      "filename": ("REQUIRED", "There MUST be exactly one row for each file."),

>       "acq_time": ("OPTIONAL", "Acquisition time refers to when the first data point in each run was acquired. Furthermore, if this header is provided, the acquisition times of all files that to a recording MUST be identical."),

Reviewer: ```suggestion
      "acq_time": ("OPTIONAL", "Acquisition time refers to when the first data point in each run was acquired. Furthermore, if this header is provided, the acquisition times of all files from the same recording MUST be identical."),
```
